tests/parse_callbacks/mod.rs

```diff
@@ -1,5 +1,29 @@
 use bindgen::callbacks::*;
 
+#[derive(Debug)]
+pub struct RemoveFunctionPrefixParseCallback {
+    pub remove_function_prefix: Option<String>,
+}
+
+impl RemoveFunctionPrefixParseCallback {
+    pub fn new(prefix: &str) -> Self {
+        RemoveFunctionPrefixParseCallback {
+            remove_function_prefix: Some(prefix.to_string()),
+        }
+    }
+}
+
+impl ParseCallbacks for RemoveFunctionPrefixParseCallback {
+    fn generated_name_override(&self, function_name: &str) -> Option<String> {
+        if let Some(prefix) = &self.remove_function_prefix {
+            if function_name.starts_with(prefix) {
```

Comment:
maybe you can use [`String::strip_prefix`](https://doc.rust-lang.org/std/string/struct.String.html#method.strip_prefix) instead?